dll/win32/netcfgx/lang/en-US.rc

```diff
@@ -7,22 +7,22 @@ FONT 8, "MS Shell Dlg"
 BEGIN
     LTEXT "You can get IP settings assigned automatically if your network supports this capability. Otherwise, you need to ask your network administrator for the appropriate IP settings.", -1, 9, 9, 235, 27
     GROUPBOX "", -1, 9, 61, 228, 60, BS_GROUPBOX
-    CONTROL "Obtain the IP address automatically", IDC_USEDHCP, "BUTTON", BS_AUTORADIOBUTTON | WS_GROUP | WS_TABSTOP, 14, 43, 210, 12
-    CONTROL "&Use the following IP address:", IDC_NODHCP, "BUTTON", BS_AUTORADIOBUTTON, 14, 59, 120, 12
-    LTEXT "IP address:", -1, 14, 75, 135, 8
+    CONTROL "&Obtain the IP address automatically", IDC_USEDHCP, "BUTTON", BS_AUTORADIOBUTTON | WS_GROUP | WS_TABSTOP, 14, 43, 210, 12
+    CONTROL "U&se the following IP address:", IDC_NODHCP, "BUTTON", BS_AUTORADIOBUTTON, 14, 59, 120, 12
```

Comment:
I do agree, but following MS is a good thing to do "by default", if nothing specific is speaking against it. Especially since it helps us to avoid introducing accelerator collisions, and allows users to persist in the workflows that may be hardwired into their brain by performing specific key-sequences for many years. But that is definitely more important in frequently used dialogs like explorer filebrowser menus than it is for netcfgx.

In this specific case MS didn't use &U here, because they already used &u within "S&ubnet mask:" 4 lines to the bottom. You see: Following MS makes our life easier. ;-)